### PR TITLE
Misc errata

### PR DESCRIPTION
**Pull request by earth2marsh**
* Removed last(?) Vendor Extension references
* Title casing on headings
* Commas, minor grammatical fixes, and small consistency issues
* Minor wordsmithing

### CODE REVIEW

File: versions/3.0.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ For example, if a field has an array value, the JSON array representation will b
 }
 ```
 
-While the API is described using JSON it does not impose a JSON input/output to the API itself.
+While the API is described using JSON, it does not impose a JSON input/output to the API itself.
```
Comment (RobDolinMS) [inline versions/3.0.md]: +1

File: versions/3.0.md [hunk 2]
```diff
@@ -311,7 +311,7 @@ url: http://www.apache.org/licenses/LICENSE-2.0.html
 
 #### <a name="serverObject"></a>Server Object
 
-An object representing a Server.
+An object representing a list of one or more servers.
```
Comment (RobDolinMS) [inline versions/3.0.md]: +1

File: versions/3.0.md [hunk 3]
```diff
@@ -757,7 +757,7 @@ Field Name | Type | Description
...
 <a name="parameterSchema"></a>schema | [Schema Object](#schemaObject) | The schema defining the type used for the parameter.
-<a name="parameterExamples"></a>examples | [[Example Object](#exampleObject)] | Examples of the content type.  Each example in the Examples array SHOULD be in the correct format as specified parameter encoding.  The `examples` object is mutually exclusive to the `example` object.  Furthermore, if referencing a `schema` which contains an example, the `examples` value SHALL _override_ the example provided by the schema.
+<a name="parameterExamples"></a>examples | [[Example Object](#exampleObject)] | Examples of the content type.  Each example in the Examples array SHOULD be in the correct format as specified parameter encoding.  The `examples` object is mutually exclusive to the `example` object.  Furthermore, if referencing a `schema` that contains an example, the `examples` value SHALL _override_ the example provided by the schema.
```
Comment (RobDolinMS) [inline versions/3.0.md]: +1

File: versions/3.0.md [hunk 4]
```diff
@@ -1783,7 +1783,7 @@ The variable expression is defined by the following [ABNF](https://tools.ietf.or
 The `name` identifier is case-sensitive, whereas `token` is not. 
 
 
-### Request parameter example
+### Request Parameter Example
```
Comment (RobDolinMS) [inline versions/3.0.md]: +1

File: versions/3.0.md [hunk 5]
```diff
@@ -1823,7 +1823,7 @@ For a `id` value of `10101110`, the generated link would be:
 href: '/users/10101110/department'
 ```
 
-### Response payload example
+### Response Payload Example
```
Comment (RobDolinMS) [inline versions/3.0.md]: +1

File: versions/3.0.md [hunk 6]
```diff
@@ -3067,7 +3067,7 @@ animals:
 #### <a name="definitionsObject"></a>Definitions Object
 
 An object to hold schemas for data types that can be consumed and produced by operations.
-These data types can be primitives, arrays or models.
+These data types can be primitives, arrays, or models.
```
Comment (RobDolinMS) [inline versions/3.0.md]: +1  Oxford comma ;)

File: versions/3.0.md (hunk 2 above)
Comment (webron) [inline versions/3.0.md]: This is not true. It represents a single server. The example for the object is wrong and needs to be updated.

**Comment by earth2marsh** [comment]
Good suggestion. Done.

**Pull request merged**